schemas/walletAddresses.js

```diff
@@ -0,0 +1,22 @@
+/**
+ * Wallet Address is a way to record additional wallet addresses for an MSA
+ * Should be stored using Itemized and Signature Required
+ * SLIP-0044 Standard: https://github.com/satoshilabs/slips/blob/master/slip-0044.md
+ */
+export default {
+  type: "record",
+  name: "Address",
+  namespace: "xyz.frequency",
+  fields: [
+    {
+      name: "coin_type",
```

Comment:
I see that SLIP-0044 calls them coin types, but should we consider the slightly more neutral "token_type" instead?

Reply:
I went back and forth on this. I am tempted toward the even more specific `slip_0044_coin_type` so that the documentation is even clearer what sort of data it is, but that's a bit much perhaps.

I want to keep it as connected as possible to the source for this data as it is otherwise only documentation that tells you how to read the data.

Perhaps I should switch to `slip_0044_coin_type` or just `slip_0044`?

Reply:
I vote slip_0044_coin_type

Reply:
Went for `coin_type_slip_0044` (so coin type is still first and easier to remember with code completion)